src/test/kotlin/org/jmailen/gradle/kotlinter/functional/CustomTaskTest.kt

```diff
@@ -22,7 +21,7 @@ class CustomTaskTest : WithGradleTest.Kotlin() {
         projectRoot = testProjectDir.root.apply {
             resolve("settings.gradle") { writeText(settingsFile) }
             resolve("build.gradle") {
-                @Language("groovy")
+                // language=groovy
```

Comment:
I also noticed that I unnecessairly added a dependency, which can be easily replaced with comments